sql/updates/world/3.3.5/2016_11_16_00_world.sql

```diff
@@ -0,0 +1,2 @@
+-- Adds a reference to the spell_vael_burning_adrenaline script for Burning Adrenaline (Vael Fight)
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES ('18173', 'spell_vael_burning_adrenaline');
```

Comment:
missing a delete line before insert (this querry should run multiple times)

something like:

``` sql
DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_vael_burning_adrenaline';
```